core/actions/data_preparation.ts

```diff
@@ -168,6 +191,22 @@ function applySessionToDataPreparationContents(
   return resolvedDataPreparation;
 }
 
+function applySessionAndDefaultsToTableReference(
```

Comment:
Why isn't this a private method on the class, rather than passing the action builder as a parameter?